TheJumpCloud/jcapi-powershell
# Sa 3037 system insights tables

## Issues
* [SA-3037](https://jumpcloud.atlassian.net/browse/SA-3037) - Add new system insights tables to SDKs and version rollup

## What does this solve?

Adds missing system Insights Tables to SDKs, update of SDK generated code

## Is there anything particularly tricky?

## How should this be tested?

Build the SDKs locally or Import Latest versions from CodeArtifact:

Import the JumpCloud PowerShell Module, you should be able to get system Insight tables for each of the newly generated endpoints:

* Get-JcSdkSystemInsightAzureInstanceMetadata.ps1
* Get-JcSdkSystemInsightAzureInstanceTag.ps1
* Get-JcSdkSystemInsightChassisInfo.ps1
* Get-JcSdkSystemInsightLinuxPackage.ps1
* Get-JcSdkSystemInsightSecureboot.ps1
* Get-JcSdkSystemInsightTpmInfo.ps1
* Get-JcSdkSystemInsightUserAssist.ps1
* Get-JcSdkSystemInsightWindowsSecurityCenter.ps1

Set-JCSdkUserAssocitaion — should no longer time out when removing a user from a system

## Screenshots
